src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -156,10 +156,6 @@ export class DictionaryModal extends React.Component {
     if (!data.owner) {
       errors.owner = 'Kindly select the Owner of the Dictionary';
     }
-    if (!data.public_access) {
```

Comment:
Could you explain how this change is related to simply changing from View, None, to Public, Private?

Reply:
By giving the Visibility select dropdown a default value, It is impossible for it to be left blank making that verification code redundant. So I did away with it. Previously, the options were only placed in a local state with an OnChange listener meaning that one had to manually make a selection for them to be added in state.